route/gin.go

```diff
@@ -40,7 +40,12 @@ func (r *Gin) Run(host ...string) error {
 			return errors.New("host can't be empty")
 		}
 
-		host = append(host, defaultHost)
+		defaultPort := facades.Config.GetString("route.port")
+		if defaultPort == "" {
+			return errors.New("port can't be empty")
+		}
+		completeHost := defaultHost + ":" + defaultPort
+		host = append(host, completeHost)
```

Comment:
Great Bro, can you also change the `RunTLS` method like this? Then we also need to change the `config/route.go` file in `goravel/goravel`.

Reply:
Yeah, I have pushed the changes, I am just try to fix failing test cases.

Reply:
Hey @hwbrzzl can you help me with testcases, I am not able to resolve them

Reply:
@hwbrzzl There is some issue with testcases, I am trying to debug it, will push it by Wednesday.